Appraisals

```diff
@@ -1,34 +1,42 @@
 appraise 'rails-3' do
+  gem 'sqlite3', "< 1.4.0"
```

Comment:
Could you use single quotes instead of double please?

Reply:
Whoops